### PR TITLE
Import force resources

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
This imports the `-f` option from scrypt; here called `--force-resources`.  This PR includes PR #214 import warning fixes from scrypt, so you probably want to merge that one first.

I also added a `--no-force-resources` option in case you wanted it, but of course it's easy to remove if you don't think somebody would want it.

To test it, I changed the hard-coded `memfrac 0.5` in `scryptdec_buf(...)` in `lib/keyfile/keyfile.c` to `memfrac 0.00001` so that my system would refuse to decrypt a keyfile, then added the flag and it worked.  I suppose I could have tried something with `ulimit`, but then the process might actually not have enough memory.

**Comment by gperciva** [comment]
Addresses #207.

**Comment by gperciva** [comment]
Closed pending a fix in scrypt.  Will re-open later.

**Comment by gperciva** [comment]
This now includes the fix from https://github.com/Tarsnap/scrypt/pull/83 (initialize memlimit and opps).

File: tar/tarsnap.1-mdoc.in [hunk 1]
```diff
@@ -368,6 +368,9 @@ containing only the entries from
 .Ar all-backup
 containing the string
 .Sq foo .
+.It Fl -force-resources
+Force the decryption to proceed even if it is anticipated to
```
Comment (cperciva) [inline tar/tarsnap.1-mdoc.in]: I think this comment needs more context, to point out that we're talking about the passphrased encryption of key files...

**Comment by gperciva** [comment]
Added a new commit to reword the man page.  Do not merge this; if you like the wording, I'll rebase the history.

(I also need to figure out why `release-tools/mdoc2man.sh` is behaving differently on Ubuntu 16.04)

**Comment by gperciva** [comment]
Updated to include PR #216 (which runs `mdoc2man.sh` which I'd forgotten about), and then run `mdoc2man.sh` to catch this latest change.

As before, let me rebase it before it gets merged.

**Comment by cperciva** [comment]
"Force the decryption of an encrypted passphrase" <-- this is meaningless.

I think you mean "Force the decryption of a passphrase-encrypted key file" ?

And I'd change "even if it is anticipated" to "even if the scrypt key derivation is anticipated".

**Comment by gperciva** [comment]
What, you don't think that people store encrypted passphrases inside tarsnap archives?  ;)   ok, revised that part.

As for the second part, as a user I'd wonder "WTM is `scrypt`?  WTM is key derivation?".

I now have:
> Force the decryption of a passphrase-encrypted key file to proceed even if it is anticipated to require an excessive amount of memory or CPU time.

and grammatically, I think the "it" refers to "decryption".

Update PR, including rebasing history.

**Pull request merged**